Formula/iconsur.rb

```diff
@@ -0,0 +1,19 @@
+require "language/node"
```

Comment:
```suggestion
require "language/node"

```
Let's try to be consistent with other formulae here.